packages/api/src/car.js

```diff
@@ -166,6 +167,11 @@ export async function handleCarUpload (request, env, ctx, car, uploadType = 'Car
     dagSize
   })
 
+  if (structure === 'Complete') {
```

Comment:
It would fail when it gets to a block or link it doesn't have index data for...which would happen at the root block for partial CARs uploaded here using carbites treewalk splitter.

The wider question is - should we still index a block if we don't have index information for all of it's links? Right now it's no.